src/main/java/ua/vedroid/cinema/model/dto/UserRegistrationDto.java

```diff
@@ -0,0 +1,49 @@
+package ua.vedroid.cinema.model.dto;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import ua.vedroid.cinema.lib.Email;
+import ua.vedroid.cinema.lib.FieldMatch;
+
+@FieldMatch(
+        first = "password",
+        second = "confirmPassword",
+        message = "The password fields must match"
+)
+public class UserRegistrationDto {
+    private static final int PASS_MIN_LENGTH = 4;
+    private static final int PASS_MAX_LENGTH = 16;
+    private static final String PASS_MESSAGE = "Password size must be between " + PASS_MIN_LENGTH
+            + " and " + PASS_MAX_LENGTH;
+    @Email
+    private String email;
+    @NotNull(message = "Field password must not be null")
+    @Size(min = PASS_MIN_LENGTH, max = PASS_MAX_LENGTH, message = PASS_MESSAGE)
```

Comment:
it's ok to write numbers right here
```suggestion
    @Size(min = 4, max = 16, message = "Password size must be between 4 and 16")
```